docs/products/clickhouse/howto/check-data-tiered-storage.rst

```diff
@@ -0,0 +1,80 @@
+Check data distribution between storage devices in Aiven for ClickHouse®'s tiered storage
+=========================================================================================
+
+.. important::
+
+    Aiven for ClickHouse® tiered storage is a :doc:`limited availability feature </docs/platform/concepts/beta_services>`. If you're interested in trying out this feature, contact the sales team at `sales@Aiven.io <mailto:sales@Aiven.io>`_.
+
+Verify how your data is distributed between the two layers of your tiered storage: SSD and object storage.
+
+About checking data distribution
+--------------------------------
+
+If you have the tiered storage feature :doc:`enabled </docs/products/clickhouse/howto/enable-tiered-storage>` on your project, your data in Aiven for ClickHouse is distributed between two storage devices (tiers). You can learn on what storage devices specific databases and tables are stored. You can also preview their total sizes as well as part counts, minimum part sizes, median part sizes, and maximum part sizes.
```

Comment:
"You can check on what storage devices your databases and tables are stored."
instead of
"You can learn on what storage devices specific databases and tables are stored."